### DISCUSSION TITLE
Can Bitcoin (or other cryptocurrencies) be used as currency for an account (cash balance)?

### DISCUSSION THREAD

**Discussion by davidecorradini** [ghostfolio/ghostfolio#3261] [Q&A]
I am not able to add Bitcoin or Ethereum as a currency to my self-hosted instance. I want to have a BTC account with the cash value in BTC. Instead, I only see that Bitcoin can be added as an investment.
Does the current implementation support such a feature?

**Comment by dtslvr** [comment]
No, this is currently not possible.

**Reply by davidecorradini** [reply]
OK, thanks for the information.
